release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3326,6 +3326,82 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-14-32_{context}"]
+=== RHSA-2024:4329 - {product-title} 4.14.32 bug fix and security updates
+
+Issued: 2024-07-11
+
+{product-title} release 4.14.32, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4329[RHSA-2024:4329] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:4332[RHBA-2024:4332] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.32 --pullspecs
+----
+
+[id="ocp-4-14-32-enhancements_{context}"]
+==== Enhancements
+
+The following enhancements are included in this z-stream release:
+
+[id="ocp-4-14-32-pipeline-trigger_{context}"]
+===== New custom resource definitions for pipeline plugin
+
+This release updates the pipelines plugin to support the latest Pipeline Trigger API version for the custom resource definitions (CRDs) `ClusterTriggerBinding`, `TriggerTemplate`, and `EventListener`. (link:https://issues.redhat.com/browse/OCPBUGS-35723[*OCPBUGS-35723*])
+
+[id="ocp-4-14-32-_{context}"]
+===== Controller for defragmentating etcd
+
+This release introduces a controller to defragment etcd for hosted clusters on Hypershift. (link:https://issues.redhat.com/browse/OCPBUGS-35290[*OCPBUGS-35723*])
+
+[id="ocp-4-14-32-insights-operator-resources_{context}"]
+===== Insights Operator collects new resources
+
+The Insights Operator (IO) now collects the `Prometheus` and `AlertManager` resources in addition to the `openshift-monitoring` custom resource. (link:https://issues.redhat.com/browse/OCPBUGS-36380[*OCPBUGS-36380*])
```

Comment:
Not sure if "IO" is approved (most things like this aren't). Since it's used one, might as well remove it imo
```suggestion
The Insights Operator now collects the `Prometheus` and `AlertManager` resources in addition to the `openshift-monitoring` custom resource. (link:https://issues.redhat.com/browse/OCPBUGS-36380[*OCPBUGS-36380*])
```